README.md

```diff
@@ -18,6 +18,8 @@ A Redsys~~Sermepa~~ payment gateway backend for [django-payments](https://github
   * Sandbox endpoint is default. Production endpoint is 'https://sis.redsys.es'
 * order_number_prefix (default:'0000'): Payment PK is suffixed to this to create Redsys order number
 * signature_version (default:'HMAC_SHA256_V1'): Only supported signature type.
+* ds_merchant_directpayment (default: "TRUE"): "TRUE" or "FALSE"
```

Comment:
Maybe will be interesting to integrate the redsys (spanish) related doc: https://pagosonline.redsys.es/oneclick.html